client/reader/discover/controller.js

```diff
@@ -12,10 +11,9 @@ const ANALYTICS_PAGE_TITLE = 'Reader';
 
 const exported = {
 	discover( context, next ) {
-		const blogId = config( 'discover_blog_id' );
 		const basePath = sectionify( context.path );
-		const fullAnalyticsPageTitle = ANALYTICS_PAGE_TITLE + ' > Site > ' + blogId;
-		const streamKey = `discover:${ blogId }`;
+		const fullAnalyticsPageTitle = ANALYTICS_PAGE_TITLE + ' > Discover';
+		const streamKey = `discover:`;
```

Comment:
we will add the suffix int he discover-stream component, since that will control whether this is recommended or a tag stream.